deploy/examples/sharedservice.json

```diff
@@ -5,10 +5,10 @@
     "name": "example"
   },
   "spec": {
-    "service_type":"keycloak",
-    "slices_per_instance":3,
-    "maximum_instances":3,
-    "minimum_instances":1,
-    "required_instances": 1
+    "serviceType":"keycloak",
+    "maxSlices":3,
+    "maxInstances":3,
+    "minInstances":1,
+    "requiredInstances": 0
```

Comment:
Set this to 0 to demonstrate that min instances will be created if this is 0. 

@philbrookes I'm assuming something else is going to bump this value as necessary?

Reply:
Aye, the requiredInstances value is going to be modified in the slice provision / deprovision handlers.